.github/workflows/github.yml

```diff
@@ -0,0 +1,66 @@
+name: GitHub
```

Comment:
Could we have a comment to explain the purpose of this GHA (like the others do) please?

Reply:
Added.